forms-flow-bpm/docker-compose-windows.yml

```diff
@@ -2,7 +2,7 @@
 version: "3.7"
 services:
   forms-flow-bpm-db:
-    image: postgres:latest
+    image: postgres:14
```

Comment:
Please check the previous comment on camunda supporting postgres 14

Reply:
It was working earlier with latest (postgress  =< 14) without any issues. So then i think it is fine to work with 14.

Reply:
I think latest actually downloads the supported version and not 14+. Until we run upgrade. Its better to go with camunda recommendations.

Reply:
https://camunda.com/blog/2021/04/camunda-platform-runtime-7150-released/ - @ambadi-aot  let us go according to the document. The point @guru-aot says is right, it is better to go with supported versions to avoid any issues.